gbpservice/nfp/configurator/agents/vpn.py

```diff
@@ -277,9 +278,9 @@ def _sync_ipsec_conns(self, context, svc_context):
         Returns: None
         """
         try:
-            self._get_driver()
 
-            return self._get_driver().check_status(context, svc_context)
+            return self._get_driver(service_vendor).check_status(context,
```

Comment:
Can we avoid invocation of _get_driver() every time in this sync logic? Can't this be maintained?